src/credentials/CHIPOperationalCredentials.h

```diff
@@ -219,6 +234,17 @@ class DLL_EXPORT OperationalCredentialSet
     CHIP_ERROR SetDevOpCred(const CertificateKeyId & trustedRootId, const uint8_t * chipDeviceCredentials,
                             uint16_t chipDeviceCredentialsLen);
 
+    /**
+     * @brief TODO Serialize the CASESession to the given serializable data structure for secure pairing
+     **/
+    CHIP_ERROR ToSerializable(const CertificateKeyId & trustedRootId, OperationalCredentialSerializable & output);
```

Comment:
Sorry, I am probably missing some context here. What is the purpose of these functions? Prepare data for storage? Where this serialization formate defined?

Reply:
Yes, prepare data for storage. Purpose is to store the Credentials in the KVS prior to performing a CASE connection. The serialization format is under struct OperationalCredentialSerializable  in the same header